### PR TITLE
[Conflicts] Optimization

### PR DESCRIPTION
**Pull request by DeShrike**
Tiny

### CODE REVIEW

File: Assets/Scripts/Controllers/BuildModeController.cs [hunk 1]
```diff
@@ -117,9 +117,9 @@ public void DoBuild(Tile t)
 
                 j.furniturePrototype = WorldController.Instance.World.furniturePrototypes[furnitureType];
 
-                for (int x_off = t.X; x_off < (t.X + WorldController.Instance.World.furniturePrototypes[furnitureType].Width); x_off++)
+                for (int x_off = t.X; x_off < (t.X + j.furniturePrototype.Width); x_off++)
```
Comment (Grenkin1988) [inline Assets/Scripts/Controllers/BuildModeController.cs]: Please change name of "j" without scrolling to the top it's not possible to know what is "j"

File: Assets/Scripts/Controllers/BuildModeController.cs [hunk 2]
```diff
@@ -230,9 +230,11 @@ public void DoBuild(Tile t)
 
     public bool DoesBuildJobOverlapExistingBuildJob(Tile t, string furnitureType)
     {
-        for (int x_off = t.X; x_off < (t.X + WorldController.Instance.World.furniturePrototypes[furnitureType].Width); x_off++)
+        Furniture furniturePrototype = WorldController.Instance.World.furniturePrototypes[furnitureType];
+
+        for (int x_off = t.X; x_off < (t.X + furniturePrototype.Width); x_off++)
         {
-            for (int y_off = t.Y; y_off < (t.Y + WorldController.Instance.World.furniturePrototypes[furnitureType].Height); y_off++)
```
Comment (Mizipzor) [inline Assets/Scripts/Controllers/BuildModeController.cs]: Could put that as a variable on the line above ` int height = WorldController.Instance.World.furniturePrototypes[furnitureType].Height`? Not only is it more readable but think expression will be check once per iteration which is a waste unless the furniture height changes.

**Comment by Grenkin1988** [comment]
Would be great to see ```public void DoBuild(Tile tile)``` instead of ```public void DoBuild(Tile t)```
but this is up to you

**Comment by Mizipzor** [comment]
Try to eliminate single character variables if you can.

**Comment by DeShrike** [comment]
I'll redo this

**Pull request closed without merging**